packages/swarm-styles/lib/components/select.css

```diff
@@ -0,0 +1,31 @@
+[data-swarm-select] {
+    display:flex;
+    /* display:block; */
+}
+
+[data-swarm-select] select {
+    -webkit-appearance: none;
```

Comment:
The prefixes aren't necessary because postcss is configured to add the necessary browser prefixes.